jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -331,6 +333,45 @@ def _parse_spectrum1d(app, file_obj, data_label=None):
     app.add_data_to_viewer('spectrum-viewer', f"{data_label}[FLUX]")
 
 
+def _parse_ndarray(app, file_obj, data_label=None, data_type=None):
+    if data_label is None:
```

Comment:
I think this part will change once #1672 is in but maybe @javerbukh can confirm.